js/index.js

```diff
@@ -0,0 +1,55 @@
+//Show message that user can choose the type of calculator.//
+alert ("Hello and welcome to Sparta-js-Calculator!")
+var chosen_calculator = prompt("Which calculator do you want to use? (b)asic and (a)dvanced");
+
+//Define all variables.
+var basic_calculator = "b";
+var advanced_calculator = "a";
+
+var basic_operation = "basic_operation";
+var advanced_operation = "advanced_operation";
+
+var additon = 1;
+var multiplication = 2;
+var division = 3;
+var subtraction = 4;
+var power = 5;
+var square = 6;
+
+//Ask the user for the numbers to operate on based on their selected calculator//
+if (chosen_calculator == "b") {
+	var basic_operation = prompt ("Which operation do you want to run? (1)Additon, (2)Multiplication, (3)Division or (4)Subtraction")
+	var first_number = prompt ("Type your first number");
```

Comment:
This gets back a string of a number, you need to parse it into an integer in order to use it in a mathematical operation.